src/active-response/utilities.c

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Don't create this file. We have already `src/active-response/active_responses.c` for common functions.